### DISCUSSION TITLE
Language problem

### DISCUSSION THREAD

**Discussion by kodaaygo** [craigk5n/webcalendar#398] [General]
Hello, after upgrade to 1.91 version, when i click Lunch WebCalendar, i've got a message:

Fatal error: Uncaught ValueError: mb_language(): Argument #1 ($language) must be a valid language, "Polish" given in /home/users/aquasfera/public_html/webcal23/includes/classes/WebCalendar.php:922 Stack trace: #0 /home/users/aquasfera/public_html/webcal23/includes/classes/WebCalendar.php(922): mb_language('Polish') #1 /home/users/aquasfera/public_html/webcal23/includes/classes/WebCalendar.php(1010): WebCalendar->_initTranslate() #2 /home/users/aquasfera/public_html/webcal23/login.php(51): WebCalendar->setLanguage() #3 {main} thrown in /home/users/aquasfera/public_html/webcal23/includes/classes/WebCalendar.php on line 922

Any ideas?

**Comment by craigk5n** [comment]
Try the latest code in master (or download master.zip) where I just put some changes in to hopefully fix this issue.
